dev/cognitiveservices/data-plane/Language/analyzeconversations.json

```
@@ -870,6 +871,26 @@
         }
       }
     },
+    "RegexKey": {
+      "description": "The regex key extra data kind.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseExtraInformation"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "RegexKey",
+      "properties": {
+        "key": {
+          "type": "string",
+          "description": "The key of the regex pattern used in extracting the entity."
+        },
+        "regexPattern": {
+          "type": "string",
+          "description": "The .NET regex pattern used in extracting the entity. For more information about .NET regular expressions, please visit (https://docs.microsoft.com/en-us/dotnet/standard/base-types/regular-expressions)."
```

Do not put locales in URIs:
```suggestion
          "description": "The .NET regex pattern used in extracting the entity. For more information about .NET regular expressions, please visit https://docs.microsoft.com/dotnet/standard/base-types/regular-expressions."
```

Also don't use parens in case this does get treated like markdown in some cases. The link may not actually show up.

---

Done.